tests/integration/test_minidump.py

```diff
@@ -464,11 +464,15 @@ def test_minidump_ratelimit(
 
     # First minidump returns 200 but is rate limited in processing
     relay.send_minidump(project_id=project_id, files=attachments)
-    outcomes_consumer.assert_rate_limited("static_disabled_quota", category="error")
+    outcomes_consumer.assert_rate_limited(
+        "static_disabled_quota", categories=["error", "attachment"]
```

Comment:
This test unveils that there is a mismatch between what `EnvelopeLimiter` does and how we generate outcomes. I'll follow up with a more detailed description.

Reply:
We will go with the current state and fine-tune this in the next stage.